src/anomalib/pipelines/benchmark/job.py

```diff
@@ -69,6 +72,7 @@ def run(
             "model": self.model.__class__.__name__,
             "data": self.datamodule.__class__.__name__,
             "category": self.datamodule.category,
+            "duration": int(duration),
```

Comment:
Do you think it would be handy to have train and test time separately? Also, if we have the individual test time then we can calculate batch throughput in the future.